src/lib/holocene/compatibility-badge.svelte

```diff
@@ -0,0 +1,32 @@
+<script lang="ts">
+  import Icon from '$lib/holocene/icon/icon.svelte';
+  import { translate } from '$lib/i18n/translate';
+
+  export let defaultVersion = false;
+  export let active = false;
+  export let buildId = '';
+</script>
+
+<p class="select-all flex gap-2 noto">
+  {#if defaultVersion && buildId}
+    <span class:active-version={active} class:version={!active}>
+      <Icon name="merge" />{buildId}
+      {#if active}{translate('workers', 'overall')}{/if}
```

Comment:
Can we pass this text in as a prop or slot to keep i18n out of Holocene?

Reply:
👍